src/components/homePage/Home.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import NavBar from '../navbar/NavBar';
+import Slideshow from './SlideShow';
+import slide from '../../assets/slide.jpg';
```

Comment:
give meaningful names to your images. What do `slide` , `slide1`, etc represent?